packages/calcite-components/src/demos/tile-group.html

```diff
@@ -569,6 +569,215 @@
         </div>
       </div>
 
+      <!-- center alignment -->
+      <div class="parent">
```

Comment:
Sidebar: it looks like the content for `tile-group` and the `allVariants` story is the same. We should look to see if there's a way of referencing this one from the story or finding a way to use storybook with the Stencil dev server.

Reply:
I've had this thought for a long while myself.  I'll look into that.

Reply:
Sounds good. Can you create a research issue and assign it to yourself to keep track of it?

Reply:
https://github.com/Esri/calcite-design-system/issues/9313